### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by wopox1337** [ChatAdditions/ChatsAdditions_AMXX#111] [General]
Tell me something

**Comment by sblockqw** [comment]
hi bro, i would like to know when ChatAdditions 1.3.0 is expected to end with a mysql connection? I already use the csbans plugin, we have top1 hungarian server with 1500-2000 players per day, I could help with testing the plugin and the web interface as well as resolving / discovering errors.

**Reply by wopox1337** [reply]
Now included in [Release v1.3.0](https://github.com/ChatAdditions/ChatAdditions_AMXX/releases/tag/v1.3.0)
